docker-compose.yml

```diff
@@ -19,10 +19,14 @@ services:
       - "./tmp:/tmp:rw"  # give permission to export the GraphQL schema here.
       - "./.docker:/.docker:ro"  # Functional Tests read files from .docker/localstack/dynamodb.
     depends_on:
-      - localstack
-      - memcached
-      - snowplow
-      - qdrant
+      localstack:
+        condition: service_started
+      memcached:
+        condition: service_started
+      snowplow:
+        condition: service_healthy  # snowplow needs to be healthy before tests can start.
+      qdrant:
+        condition: service_started
```

Comment:
`service_started` is the default. [Docker Compose depends_on documentation](https://docs.docker.com/compose/compose-file/05-services/#depends_on) states that this is the default in the short syntax that was used before.